service/src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -822,9 +829,14 @@ public void cloneGcpWorkspace() {
         controlledResourceService.createControlledResourceSync(
             bucketResource, ControlledResourceIamRole.OWNER, USER_REQUEST, creationParameters);
 
-    // Enable an application.
+    // Enable an application
     appService.enableWorkspaceApplication(USER_REQUEST, sourceWorkspace, TEST_WSM_APP);
 
+    // Create a folder
+    UUID folderId = UUID.randomUUID();
+    folderDao.createFolder(
+        new Folder(folderId, sourceWorkspaceId, PARENT_FOLDER_NAME, null, null, null));
```

Comment:
nit: comment for the null field and pass in empty map for properties. https://cs.github.com/DataBiosphere/terra-workspace-manager/blob/fed3185e5babac58eebf9dc90d5eaab32a9156f5/service/src/main/java/bio/terra/workspace/service/folder/model/Folder.java?q=folder+path%3Amodel#L13